js/classes/entities/foreground/ui/start_menu.js

```diff
@@ -75,6 +75,9 @@ class StartMenu {
         GAME.mode = GameEngine.GAMEPLAY_MODE;
         HUD.addComponents();
         this.removeFromWorld = true;
+        // setTimeout(() => {
+        //     ASSET_MGR.playMusic(MUSIC.PEACEFUL_CHIPTUNE.path, MUSIC.PEACEFUL_CHIPTUNE.volume);
+        // }, 1000);
```

Comment:
Why is this commented out?

Reply:
It was a plan B, but I believe I can delete it now